vmdb/app/models/ems_cluster.rb

```diff
@@ -317,7 +317,7 @@ def self.get_perf_collection_object_list
     end
 
     hids = cl_hash.collect do |a|
-      k, v = a
+      _, v = a
```

Comment:
This can be simplified better by putting it into the .collect params...

```ruby
hids = cl_hash.collect { |_k, v| v[:ho_ids] }.flatten.compact.uniq
```
```